harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -1235,6 +1236,21 @@ def test_trial_validation_checkpointing(self, tmp_path: pathlib.Path):
             controller.core_context.train.get_experiment_best_validation.reset_mock()
             controller._checkpoint.reset_mock()
 
+    @mock.patch.object(det.core.DummySearcherOperation, "report_progress")
+    def test_searcher_progress_reporting(self, mock_report_progress: mock.MagicMock):
+        trial, controller = pytorch_utils.create_trial_and_trial_controller(
+            trial_class=pytorch_onevar_model.OneVarTrial,
+            scheduling_unit=10,
+            hparams=self.hparams,
+            trial_seed=self.trial_seed,
+            max_batches=100,
+        )
+        controller.run()
+        assert mock_report_progress.call_count == 100 // 10
```

Comment:
this line isn't necessary with the list comparision.  In fact, it will just make it harder to debug if you see a failure in the logs.